src/konnektor/network_planners/generators/clustered_network_generator.py

```diff
@@ -51,19 +51,19 @@ def __init__(
             The algorithm works as follows:
             1. Cluster `Component` s with the `clusterer` obj.
             2. Build sub-networks in the clusters using the `sub_network_planners`.
-            3. Concatenate all sub-networks using the `concatenator` in order to build the final network.
+            3. Concatenate all sub-networks using the `concatenator` to build the final network.
 
 
         Parameters
         ----------
-        clusterer: ComponentsDiversityClusterer
-            This class is seperating the `Component` s along the first dimension.
         sub_network_planners: Iterable[NetworkGenerator]
-            The clusters, are then seperatley translated to sub networks by the sub_network_planners
+            NetworkGenerator(s) used to translate clusters to sub-networks.
         concatenator: NetworkConcatenator
-            The `NetworkConcatenator` is connecting the different sub networks.
-        mapper:  Union[AtomMapper, list[AtomMapper]]
-            the atom mapper is required, to define the connection between two ligands, if only `NetworkConcatenator` or  `NetworkGenerator` classes are passed
+            A `NetworkConcatenator` used to connect sub-networks.
+        clusterer: ComponentsDiversityClusterer
+            Separates the `Component` s along the first dimension.
+        mappers:  Union[AtomMapper, list[AtomMapper]]
+            Defines the connection between two ligands if `NetworkConcatenator`s or  `NetworkGenerator`s are provided. Otherwise, (?) (default:None)
```

Comment:
```suggestion
            Defines the connection between two ligands if `NetworkConcatenator` s or  `NetworkGenerator` s are provided. Otherwise, (?) (default:None)
```